scripts/oecd/regional_demography/deaths/preprocess_csv_test.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import os
+import pandas as pd
+import tempfile
+import sys
+
+MODULE_DIR = os.path.dirname(__file__)
+sys.path.insert(0, MODULE_DIR)
+from preprocess_csv import process_data, generate_tmcf
+
+sys.path.insert(1, os.path.join(MODULE_DIR, '../'))
+from download import download_data_to_file_and_df
+
+TEST_DATASET_DIR = os.path.join(MODULE_DIR, "test_data", "sample_data")
+
+EXPECTED_FILES_DIR = os.path.join(MODULE_DIR, "test_data", "expected_files")
+
+
+class TestProcess(unittest.TestCase):
+    """
+    TestPreprocess is inherting unittest class
+    properties which further requried for unit testing.
+    The test will be conducted for EuroStat BMI Sample Datasets,
+    It will be generating CSV, MCF and TMCF files based on the sample input.
+    Comparing the data with the expected files.
+    """
+
+    def __init__(self, methodName: str = ...) -> None:
```

Comment:
pls use setUpClass instead: https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUpClass
```
@classmethod
def setUpClass(cls):
   with ...
```

Reply:
used setUpClass instead of __init__